README.adoc

```diff
@@ -192,8 +221,28 @@ server.xml
 include::finish/src/main/liberty/config/server.xml[tags=**;!comment]
 ----
 
+== Building and running the application
+
+To build the application, run the Maven `install` phase from the command line in the `start` directory:
```

Comment:
FYI, there's also the common adoc for building and running the app: https://github.com/OpenLiberty/guides-common/blob/master/mvnbuild.adoc. Looks like it can be used, instead of rewriting the text.